README.md

```diff
@@ -5,12 +5,12 @@
 </p>
 
 <p align="center">
+      <p>Fundamental-Vue is a light-weight Vue.JS component library for [SAP Fiori Fundamentals](https://github.com/SAP/fundamental).</p>
```

Comment:
maybe change Vue.JS to Vue.js for consistency?

Reply:
I thought I did that... hrm :D thanks for catching that one.